webapp/src/widgets/menuWrapper.tsx

```diff
@@ -85,6 +85,7 @@ const MenuWrapper = React.memo((props: Props) => {
             className={className}
             onClick={toggle}
             ref={node}
+            data-testid='menuwrapper'
```

Comment:
Where is this attribute being used and what for?

Reply:
This attribute is used in test only and don't have consequence in render
more info : https://testing-library.com/docs/queries/bytestid

Reply:
I would prefer to set the `role` and `aria-label` rather than `data-testid`. 
See [priority] (https://testing-library.com/docs/queries/about#priority)